src/Codacy/Coverage/Parser/CloverParser.php

```diff
@@ -64,7 +64,7 @@ private function makeFileReports(\SimpleXMLElement $node)
      */
     private function makeFileReportsFromFiles(\SimpleXMLElement $node, $fileReports)
     {
-        foreach ($node as $file) {
+        foreach ($node->file as $file) {
```

Comment:
Could you explain the reason for the change? Also it would help if you provide your clover report file.